README.md

```diff
@@ -1 +1,71 @@
-# linter-configuration
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Set up a "Hello Microverse" project
+
+## Built With
+
+- Html
+- Css
+
+## Getting Started
+*Clone Your Github Repository*
+Follow these steps to clone your Github repo on Windows:
+1. Open Git Bash;
+- If Git is not already installed, it is super simple. Just go to the Git Download Folder and follow the instructions.
+2. Go to the current directory where you want the cloned directory to be added;
+- To do this, input cd and add your folder location. You can add the folder location by dragging the folder to Git bash.
+- $ cd '/c/Users/Documents/My First Git Project'
+3. Go to the page of the repository that you want to clone.
+4. Click on “Clone or download” and copy the URL.
```

Comment:
To have a clear and readable README file, please remove these unnecessary lines.